packages/zoe/zoe.chainmail

```diff
@@ -140,41 +133,33 @@ interface ZoeGoverningContractFacet () {
    * this to mint liquidity tokens and add them to the rights managed
    * by Zoe.
    */
-  escrowOffer (payoutRules :List(PayoutRulesElem), offerPayments :List(Payment)) -> (offerHandle);
-
-  /** Burn and validate an escrowReceipt received from the user. */
-  burnEscrowReceipt (escrowReceipt :Payment) -> (Extent);
```

Comment:
The use of .burnEscrowReceipt() is still reflected in the zoe guide readme. Furthermore, the zoe guide readme also refers to the publiceSwap.js file, which has been deleted. Is there literature for an updated zoe guide?

Reply:
I think the task that we need help with is updating the zoe guide readme, so we don't have more updated documentation written, but I would be happy to help explain.